cmd/start/start.go

```diff
@@ -144,15 +149,16 @@ func startZitadel(config *Config, masterKey string, server chan<- *Server) error
 	}
 
 	config.Eventstore.Pusher = new_es.NewEventstore(esPusherDBClient)
-	config.Eventstore.Querier = old_es.NewCRDB(zitadelDBClient)
+	config.Eventstore.Querier = old_es.NewCRDB(queryDBClient)
```

Comment:
isn't there a problem since the eventstore querier will also be used inside the projections to retrieve additional events, which is one of the reasons for a potential deadlock, right?